pwnlib/elf/corefile.py

```diff
@@ -945,16 +945,11 @@ def _parse_stack(self):
 
         # Sanity check!
         try:
-            assert stack[address] == b'\x00'
-        except AssertionError:
-            # Something weird is happening.  Just don't touch it.
-            log.debug("Something is weird")
-            return
+            if stack[address] == b'\x00':
```

Comment:
@peace-maker is right
```suggestion
            if stack[address] != b'\x00':
```